frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -41,7 +45,13 @@ interface IManagePoliciesPageProps {
     hash: string;
     key: string;
     pathname: string;
-    query: { team_id?: string };
+    query: {
```

Comment:
Would it make sense to call this field `queryParams`? That would both disambiguate it from its sub-field of the same name and avoid having to use that intermediary variable on line 65?